applications/FluidDynamicsApplication/custom_elements/surface_smoothing_element.h

```diff
@@ -0,0 +1,375 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Mohammad R. Hashemi
+//
+
+#if !defined(KRATOS_SURFACE_SMOOTHING_ELEMENT_H_INCLUDED )
+#define KRATOS_SURFACE_SMOOTHING_ELEMENT_H_INCLUDED
+
+// System includes
+#include <string>
+#include <iostream>
+
+// External includes
+
+// Project includes
+#include "includes/element.h"
+#include "utilities/element_size_calculator.h"
+#include "includes/deprecated_variables.h" //For IS_STRUCTURED
```

Comment:
this should not be used in new code
I think you are not even using it

Reply:
Hi @philbucher 
Thanks for reviewing.

I used the deprecated_variables IS_STRUCTURED. Other option is to introduce a new variable (within fluid dynamics application) with the same meaning (sth like ON_SURFACE). Flags would also work fine but I could not find anything relevant. Please comment if you have any idea.

Reply:
@rubenzorrilla do you know a suitable example for using `Flags`?

Reply:
> @rubenzorrilla do you know a suitable example for using `Flags`?

I meant a flag with the relevant meaning. I checked the list of flags and could not find sth. meaningful.

Reply:
The number of flags is limited because they are implemented as a concatenation of bit values, meaning that the number of flag "slots" is limited.

I agree @philbucher that no deprecated feature should be used. If possible, Maybe `BOUNDARY` or `CONTACT`can do the job. If you think this is still misleading, I'd go for a new variable in the `FluidDynamicsApplication`, but note that this option will introduce a memory overhead that will not appear if using a flag.

Reply:
Hi @rubenzorrilla 
Thank you for the hint. 
So, I will use CONTACT all over my solver.